src/xdocs/config_imports.xml

```diff
@@ -1671,7 +1714,6 @@ import java.util.stream.IntStream;
   &lt;property name=&quot;sortStaticImportsAlphabetically&quot; value=&quot;true&quot;/&gt;
 &lt;/module&gt;
         </source>
-
```

Comment:
revert this

Reply:
done

Reply:
done

> Test is failing because contents of javadoc and xml xdoc are different

omg who made this rule, i m gonna sue him